internal/apps/dop/providers/projectpipeline/service_impl.go

```diff
@@ -415,19 +415,15 @@ func (p *ProjectPipelineService) createCronIfNotExist(definition *dpb.PipelineDe
 	if err != nil {
 		return err
 	}
+	if len(crons.Data) == 1 && crons.Data[0].PipelineDefinitionID == definition.ID {
```

Comment:
how about only one cron record but definition id is not equal?

Reply:
recreate the pipeline